### PR TITLE
[metricbeat]kube scheduler metricset

### PR DESCRIPTION
**Pull request by odacremolbap**
Add `kubescheduler` metricset for `kubernetes` module

### CODE REVIEW

**Comment by odacremolbap** [comment]
jenkins, test this please

**Comment by odacremolbap** [comment]
jenkins, test this

File: metricbeat/module/kubernetes/scheduler/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Version history
+
+- June 2019, `v1.14.0`
+
+## Resources
+
+https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/metrics/metrics.go
+
+## Metrics insight
```
Comment (kaiyan-sheng) [inline metricbeat/module/kubernetes/scheduler/README.md]: hmmm I'm not sure if this is intended, the formatting here seems a bit off.

File: metricbeat/module/kubernetes/scheduler/_meta/fields.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    description: >
+      Request host
+  - name: name
+    type: keyword
+    description: >
+      Name for the resource
+  - name: result
+    type: keyword
+    description: >
+      Schedule attmpt result
```
Comment (jsoriano) [inline metricbeat/module/kubernetes/scheduler/_meta/fields.yml]:
```suggestion
      Schedule attempt result
```

File: metricbeat/module/kubernetes/scheduler/_meta/fields.yml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    description: >
+      Whether the node is master
+
+  - name: scheduler
+    type: group
+    fields:
+      - name: e2e.duration.us.bucket.*
+        type: object
+        object_type: long
+        description: E2e scheduling duration microseconds
```
Comment (jsoriano) [inline metricbeat/module/kubernetes/scheduler/_meta/fields.yml]: What is e2e here? end to end? maybe it worths to write here in the description the non-abbreviated words.

**Comment by odacremolbap** [comment]
jenkins, test this

**Pull request merged**